packages/node_modules/@webex/internal-plugin-llm/src/llm.ts

```diff
@@ -0,0 +1,114 @@
+// @ts-ignore
```

Comment:
do you now what fails if you enable ts lint here ?

we want atleast the main class to have types which we export to developers / web client

Reply:
removed